tiledb/api/c_api/attribute/test/CMakeLists.txt

```diff
@@ -28,5 +28,6 @@ include(unit_test)
 
 commence(unit_test capi_attribute)
   this_target_sources(unit_capi_attribute.cc)
+  this_target_object_libraries(capi_string)
```

Comment:
I think this should be moved to `capi_attribute_stub`.

Reply:
Theo's right. The string library should be on the object library.